src/_rules/outline.js

```diff
@@ -0,0 +1,20 @@
+import { lineWidth } from '#theme';
+
+const outlineNone = {
+  outline: '2px solid transparent',
+  'outline-offset': '2px',
+};
+
+export const outlineStyle = [
+  ["outline-none", { ... outlineNone }],
+  ["outline", { "outline-style": "solid" }],
+  ["outline-dashed", { "outline-style": "dashed" }],
+  ["outline-dotted", { "outline-style": "dotted" }],
+  ["outline-double", { "outline-style": "double" }],
+];
+export const outlineWidth = [
+  [/^outline-(\d+)$/, ([, w]) => ({ ['outline-width']: lineWidth?.[w ?? 1] }), { autocomplete: `(outline)-(${Object.keys(lineWidth).join('|')})` }],
```

Comment:
I think outline doesn't need to be in `()` in the autocomplete section because it's just one string, no other options within that group to match. Docs on unocss' autocomplete: https://github.com/unocss/unocss/blob/main/docs/tools/autocomplete.md

Reply:
https://github.com/warp-ds/drive/pull/125/commits/57b14f9397bc6a89446c78354aad71d8530c0009